cmd/eudico-stats/main.go

```diff
@@ -0,0 +1,138 @@
+package main
```

Comment:
This file is a copy-past of lotus_stats, right? Are we collecting any metrics from Lotus' default views?

Reply:
Should be. Based on my current understanding, as long as the `stats.Record` is invoked, it is exported to prometheus. I also didnt see the Lotus default view in the `lotus-stats`, I'll check again when I add the HC metrics.